tests/e2e/Services/Functions/FunctionsCustomClientTest.php

```diff
@@ -78,14 +78,17 @@ public function testCreateExecution():array
             'x-appwrite-project' => $this->getProject()['$id'],
             'x-appwrite-key' => $this->getProject()['apiKey'],
         ], [
-            'command' => 'php index.php',
+            'entrypoint' => 'index.php',
             'code' => new CURLFile(realpath(__DIR__ . '/../../../resources/functions/php.tar.gz'), 'application/x-gzip', 'php-fx.tar.gz'),
         ]);
 
         $tagId = $tag['body']['$id'] ?? '';
 
         $this->assertEquals(201, $tag['headers']['status-code']);
 
+        // Wait for tag to be built.
+        sleep(5);
```

Comment:
How secure is this? Wouldn't tests fail randomly if the server is slow?

Reply:
Since we are using the PHP runtime for testing with no `composer.json` in the test functions all the build stage does is grab the runtime server's own dependencies and tarball it. I can't see this taking more that 5 seconds even on slow servers.